pages/common/mutool.md

```diff
@@ -0,0 +1,23 @@
+# mutool
+
+> Convert PDF files, query information and extract data.
+
+- Convert pages 1-10 into 10 PNG images:
+
+`mutool convert -o {{image%d.png}} {{file.pdf}} {{1-10}}`
+
+- Convert pages 2, 3 and 5 of a PDF into text in the standard output:
+
+`mutool draw -F {{txt}} {{file.pdf}} {{2,3,5}}`
+
+- Concatenate all pages skipping the first from two PDFs into a new one:
+
+`mutool merge -o {{output.pdf}} {{input1.pdf}} 2-N {{input2.pdf}} 2-N`
+
+- Query information about all content embedded in a PDF:
+
+`mutool info {{input.pdf}}`
+
+- Extract all images, fonts and resources embedded in a PDF in the current directory:
```

Comment:
Perhaps ...`in a PDF out into the current`..... would read better?

Reply:
Yep, it would!